tests/test_api.py

```diff
@@ -515,7 +515,7 @@ def test_linked_thumbnail_url(self):
 
         else:
             expected_version_with_project = {
-                'code': 'Sg unittest version',
+                'code': self.config.version_code,
```

Comment:
The CI sandboxes entities per CI user, so we should be getting the name from the configuration and not a hardcoded value.